src/components/search/searchfields.js

```diff
@@ -43,7 +53,7 @@ define(['layoutManager', 'globalize', 'require', 'events', 'browser', 'alphaPick
 
     function initAlphaPicker(alphaPickerElement, instance) {
 
-        instance.alphaPicker = new AlphaPicker({
+        instance.alphaPicker = new AlphaPicker.default({
```

Comment:
```suggestion
        instance.alphaPicker = new AlphaPicker({
```
Looks like `.default` is not required to import ES6 into ES6.